test/utils/paths.go

```diff
@@ -50,8 +50,11 @@ func RootDir() (string, error) {
 	path := filepath.Dir(testExec)
 
 	// Look for the kubernetes source root directory
-	if strings.Contains(path, "k8s.io/kubernetes") {
-		splitPath := strings.Split(path, "k8s.io/kubernetes")
+	if strings.Contains(path, "kubernetes") {
```

Comment:
this function has lots of callers, is this change reasonable for all of them?

Reply:
a single `kubernetes` segment is much less of a signal than the fully-qualified component name. should we look for something more indicative like the name *and* a source file known to be in the root?

Reply:
if we're going to look for this we can check for something more substantial like the go.mod specifying that it is the k8s.io/kubernetes module

Reply:
> a single `kubernetes` segment is much less of a signal than the fully-qualified component name. should we look for something more indicative like the name _and_ a source file known to be in the root?

I agree with @BenTheElder  that we can't judge by name alone. Besides, do we lack some documentation about these situations?